core/src/main/java/com/netflix/conductor/core/events/ScriptEvaluator.java

```diff
@@ -90,4 +105,18 @@ public static Boolean toBoolean(Object input) {
         }
         return false;
     }
+
+    /**
+     * Validates the script for security risks.
+     *
+     * @param script Script to be validated.
+     * @throws ScriptException
+     */
+    private static void validateScript(String script) throws ScriptException {
+        for (String pattern : FORBIDDEN_PATTERNS) {
```

Comment:
Hi, this does nothing against obfuscations.
At the moment in OSS script execution it is assumed that code is trusted.
I would accept "--no-java" option for nashorn engine init though, because it is likely that we will apply it this year anyway.